assets/js/stripe-admin.js

```diff
@@ -86,6 +88,25 @@ jQuery( function( $ ) {
 					} );
 				} );
 			} );
+
+			// Webhook verification checks for timestamp within 5 minutes so warn if
+			// server time is off from browser time by > 4 minutes.
+			var timeDifference = Date.now() / 1000 - wc_stripe_settings_params.time;
+			var isTimeOutOfSync = Math.abs( timeDifference ) > 4 * 60;
+			$( '#woocommerce_stripe_test_webhook_secret, #woocommerce_stripe_webhook_secret' )
+				.on( 'change input', function() {
+					var $td = $( this ).closest( 'td' );
+					var $warning = $td.find( '.webhook_secret_time_sync_warning' );
+
+					if ( $( this ).val() ){
+						if ( isTimeOutOfSync && ! $warning.length ) {
```

Comment:
Could also call out the meaning of `$warning.length` using a variable like `hasWarningAlready` etc

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1226/commits/46a4f40f43709f934f86c2a7997e259a98a6b2ba